evolver/calibration/interface.py

```diff
@@ -150,3 +169,57 @@ class Config(Calibrator.Config):
 
         input_transformer: dict[int, ConfigDescriptor | Transformer | None] | None = None
         output_transformer: dict[int, ConfigDescriptor | Transformer | None] | None = None
+
+    def initialize_calibration_procedure(self, *args, **kwargs): ...
+
+
+# Example calibrator implementation to match existing temperature calibration flow.
+class TemperatureCalibrator(Calibrator):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.state = {"selected_vials": []}
```

Comment:
we can always do this later if desired, but I wonder if like other things we do in the framework, that we would like state to be a pydantic model?

Reply:
Ok yeah I'll implement that in a seperate PR. Maybe once you've added the CalibrationData class to the Calibrator? Since that's where the state here ends up...